themes/iterate/iterate.theme.bash

```diff
@@ -52,8 +55,8 @@ function prompt_command() {
 	fi
 
 	local wrap_char=""
-	[[ $COLUMNS && ${#new_PS1} > $(($COLUMNS / 1)) ]] && wrap_char="\n"
-	PS1="${new_PS1}${green}${wrap_char}→${reset_color} "
+	[[ $COLUMNS && ${#new_PS1} -gt $((COLUMNS / 1)) ]] && wrap_char="\n"
```

Comment:
What does `COLUMNS / 1` do? Couldn't this be just `COLUMNS`? The operands of `-gt` are subject to the arithmetic evaluation, so one doesn't need to use the arithmetic expansion `$(())` explicitly.
```suggestion
	[[ $COLUMNS && ${#new_PS1} -gt COLUMNS ]] && wrap_char="\n"
```

Reply:
> -gt COLUMNS

or maybe -gt ${COLUMNS}

Reply:
> The operands of -gt are subject to the arithmetic evaluation, so one doesn't need to use the arithmetic expansion $(()) explicitly.

TIL!.